.readme-partials.yaml

```diff
@@ -72,20 +72,14 @@ custom_content: |
   }
   ```
 about: |
-  [Google Pub/Sub Lite][product-docs] is designed to provide reliable,
-  many-to-many, asynchronous messaging between applications. Publisher
-  applications can send messages to a topic and other applications can
-  subscribe to that topic to receive the messages. By decoupling senders and
-  receivers, Google Cloud Pub/Sub allows developers to communicate between
-  independently written applications. See the
-  [Google Pub/Sub Lite docs](https://cloud.google.com/pubsub/quickstart-console#before-you-begin)
-  for more details on how to activate Pub/Sub Lite for your project, as well as
-  guidance on how to choose between Cloud Pub/Sub and Pub/Sub Lite.
+  [Google Cloud Pub/Sub Lite][product-docs] is a zonal, real-time messaging
+  service that lets you send and receive messages between independent
+  applications. You can manually configure the throughput and storage capacity
+  for Pub/Sub Lite systems.
```

Comment:
I think systems is redundant

Reply:
https://cloud.google.com/pubsub/lite/docs/

Reply:
I inherited this from the product page. Let me bring it up with the TW too. In the meanwhile, this might be okay?

Reply:
This is fine if its just a copy of documentation from elsewhere.